src/Components/Home/ContributionSection.js

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import { Text, View, Image } from 'react-native';
+import styles from './styles';
+import SectionHeader from '../SectionHeader';
+
+
+function ContributionSection() {
+    return (
+        <View>
+        <SectionHeader title= "INTERESTED IN COMPUTING" />
+        <Text style={styles.description}>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed</Text>
+        <View style={styles.container}>
+        <View style={styles.card}>
+        <Image
+        source= {require('./../../assets/contribution/code.png')}
+        style={styles.contributionImage}
+        resizeMode="contain"
+        />
+        <Text style={styles.imageText}>CODE</Text>
+        </View>
+        
+        <View style={[styles.card, styles.cardMiddle]}>
+        <Image
+        source= {require('./../../assets/contribution/documentation_training.png')}
+        style={styles.contributionImage}
+        resizeMode="contain"
+        />
+        <Text style={styles.imageText}>DOCUMENTATION/</Text>
+        <Text style={styles.imageText}>TRAINING</Text>
+        </View>
+        
+        
+        <View style={[styles.card, styles.cardLast]}>
+        <Image
+        source= {require('./../../assets/contribution/design_community.png')}
+        style={styles.contributionImage}
+        resizeMode="contain"
+        />
+        <Text style={styles.imageText}>DESIGN</Text>
+        </View>
+        
+        <View style={styles.card}>
+        <Image
+        source= {require('./../../assets/contribution/testing.png')}
+        style={styles.contributionImage}
+        resizeMode="contain"
+        />
+        <Text style={styles.imageText}>TESTING</Text>
+        </View>
+        
+        <View style={[styles.card, styles.cardMiddle]}>
+        <Image
+        source= {require('./../../assets/contribution/research_outreach.png')}
+        style={styles.contributionImage}
+        resizeMode="contain"
+        />
+        <Text style={styles.imageText}>OUTREACH/RESEARCH</Text>
+        </View>
+        
+        
+        <View style={[styles.card, styles.cardLast]}>
+        <Image
+        source= {require('./../../assets/contribution/project_management.png')}
+        style={styles.contributionImage}
+        resizeMode="contain"
+        />
+        <Text style={styles.imageText}>PROJECT MANAGEMENT</Text>
+        </View>
+        </View>
+        </View>
+        );
+    }
+    
+    
+    export default ContributionSection;
```

Comment:
you can remove this {export default ContributionSection as it is of no use after  the program is written.

Reply:
But later we would need to use it right so why keep it as a small task  to add export again?

Reply:
this export is good @19-pragya